scripts/config_osm_data.py

```diff
@@ -161,25 +161,27 @@
 
 world_iso = {
     "Africa": {
-        "DZ": "algeria",
+       "DZ": "algeria",
         "AO": "angola",
         "BJ": "benin",
         "BW": "botswana",
+        # "IO": "british-indian-ocean-territory", # Island
         "BF": "burkina-faso",
         "BI": "burundi",
         "CM": "cameroon",
-        # canary-islands    # Island
+       	// # canary-islands    # Island
         # "CV": "cape-verde", # Island
         "CF": "central-african-republic",
         "TD": "chad",
-        # "KM": "comores", # Island
-        "CG": "congo-brazzaville",
+        # "KM": "comoros", # Island
+       	"CG": "congo-brazzaville",
         "CD": "congo-democratic-republic",
         "DJ": "djibouti",
         "EG": "egypt",
         "GQ": "equatorial-guinea",
         "ER": "eritrea",
         "ET": "ethiopia",
+	    "TF": "french-southern-territories",
```

Comment:
Harmonize the data, there is here an unnecessary tab; this issue happens also below